src/lib/HST/Frontend/Transformer/Messages.hs

```diff
@@ -24,3 +26,142 @@ skipNotSupported
 skipNotSupported feature = report
   $ Message Info
   $ feature ++ " are not supported and will be skipped!"
+
+-- | Reports a fatal error that the given feature is not supported. Displays an
+--   excerpt of the input code specified by the given source span.
+notSupportedWithExcerpt
```

Comment:
Are the old functions still used?

Reply:
You mean `notSupported` and `skipNotSupported`? Yes, both are used. `notSupported` is used for very few locations where we don't have a source span or where we would display the entire module. `skipNotSupported` hasn't yet been replaced by `skipNotSupportedWithExcerpt` at all, actually. Look at the pull request description for more details and give me your opinion if you want.

Reply:
The old functions have now been removed and the new functions have gotten their names.